extensions/wikia/WallNotifications/WallNotificationEntity.class.php

```diff
@@ -1,61 +1,116 @@
 <?php
 
-class WallNotificationEntity {
+use Wikia\Logger\WikiaLogger;
 
+class WallNotificationEntity {
+	/** @var int */
 	public $id;
```

Comment:
I have no idea why the data in ```$data_noncached``` is not supposed to be cached (and ```$data_non_cached``` isn't used at all) but most of the data stored in ```$data_noncached``` isn't used anywhere, so I declared the few that are as real class members instead since they don't need to be handled in bulk (as ```$data``` does)